### PR TITLE
Signup via Exact Target

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
@ScottDowne r?

**Comment by ScottDowne** [comment]
@jbuck Merge conflict, but, I think this is fine. Let's get it up to staging and test.

**Comment by ScottDowne** [comment]
The merge conflict is due to the `value="true">` change on a line that had some temp english strings.

File: server.js [hunk 1]
```diff
@@ -12,85 +16,76 @@ server.connection({
...
+      handler: function(request, reply) {
+        reply.redirect('/data-retention/');
+      }
+    }, {
+      method: 'POST',
+      path: '/api/signup',
+      config: {
+        validate: {
+          payload: {
+            petition: Joi.string().equal(campaigns),
```
Comment (alicoding) [inline server.js]: Is this suppose to be campaign name instead of the whole object?

Comment (alicoding) [inline server.js]: Ignore me

**Comment by alicoding** [comment]
R+

**Pull request merged**